### PR TITLE
added a book

### PR DESCRIPTION
**Pull request by tonyxrandall**
**Notice:** Due to the upcoming holidays, PR reviews will be slower than usual. We hope that you have a wonderful holiday season. Happy 2022! – The Flutter Team

---

_Description of what this PR is changing or adding, and why:_

_Issues fixed by this PR (if any):_

## Presubmit checklist
- [ ] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [ ] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [ ] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by khanhnwin** [comment]
Hi @tonyxrandall! Thanks for this PR! Can you please sign the Google CLA? I'll re-run the tests once it's bee completed.

**Comment by tonyxrandall** [comment]
> Hi @tonyxrandall! Thanks for this PR! Can you please sign the Google CLA? I'll re-run the tests once it's bee completed.

done!

File: src/_data/books.yml [hunk 1]
```diff
@@ -85,10 +85,17 @@
   cover: practical-flutter.jpg
   link: "https://www.apress.com/us/book/9781484249710"
   desc: "Improve your Mobile Development with Google's latest open source SDK."
+ - title: 'Pragmatic Flutter'
```
Comment (domesticmouse) [inline src/_data/books.yml:88]: The indenting on this line looks wrong, and the linter is upset as well.

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
PTAL @sfshaza2

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Pull request merged**